sei_dbt/README.md

```diff
@@ -79,17 +108,155 @@ Then, it will:
          path: "path/to/your/database.db"  # Path to your DuckDB database file
          extensions: ['parquet', 'json']   # Optional extensions to load, if needed
             duckdb_project:
+               threads: 4
+         external_root: "data/dev"
 
        test:
          type: duckdb
          path: "test.db"  # Test database path
+         external_root: "data/test" 
          
        prod:
          type: duckdb
          path: "prod.db"  # Production database path
+         external_root: "data/prod"
    ```
+   
+In this `profiles.yml`:
+- We have three environments: `dev`, `test` and `prod`.
+- For each environment, we define:
+  - `external_root`: A variable points to the directory where CSV files or other data are located.  
+  - `dbt-duckdb` only support certain keys in the `profiles.yml`; for example, `external_root`, `settings`, and `config_options`, etc, and each has its own purpose. `custom`, `vars` are not supported. 
+   
+#### Key Usage:
+- **Global Connection Configuration**: You use `profiles.yml` to store environment-specific details such as database credentials, which will be applied to all dbt projects that use this profile.
+- **Sensitive Information**: The `profiles.yml` file is the right place to store sensitive information like database credentials, API tokens, or connection settings. It is generally **not version-controlled**, as it may contain secrets.
+- **Multiple Environments**: You can define multiple environments (like `dev`, `prod`, `test`) within `profiles.yml` and switch between them using the `target` option. This is useful when different teams or environments need to connect to different databases or use different credentials.
+
+#### Default Location:
+- The `profiles.yml` file is located by default in `~/.dbt/profiles.yml`, making it global across multiple dbt projects on your local machine. If you use a Windows machine, it could be in `C:\Users\myusername\.dbt`.
+- You can also override this by explicitly specifying the location of `profiles.yml` with the `DBT_PROFILES_DIR` environment variable.
+
+
+To **infer variables from `profiles.yml`** in your **dbt model file**, you need to use **Jinja** templating and the `target` object, which exposes information from `profiles.yml`.
+
+
+#### Step 2: Access the Variables in the Model File
```

Comment:
We went from numbered list to step 2 here, not sure how that happened.

Reply:
Yes. The first step is removed, so we will remove the second step. Just keep it as one step.